test/integration/testImage/build.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+REGISTRY=sonobuoy
+TARGET=testimage
+IMAGE_VERSION=v0.1
+
+build(){
+	docker build \
+    	-t $REGISTRY/$TARGET:$IMAGE_VERSION \
+    	-f Dockerfile \
+		.
+}
+
+build
```

Comment:
Converted the tiny makefile here to a tiny script as well. No reason not to really. Just being consistent.